peepingtom/core/datablocks/points.py

```diff
@@ -147,7 +147,7 @@ def as_zyx(self):
             return data
 
     @property
-    def n_points(self):
+    def n(self):
```

Comment:
I meant this!

Reply:
hmm, I changed it because just n isn't at all descriptive and forces you to look at the docstring